.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,54 @@
+name: Backend Pull Request
+
+on:
+  pull_request:
+    branches:
+      - '*'
+
+permissions:
+  read:
+    - actions
+    - checks
+    - contents
+    - pull-requests
+  write:
+    - checks
+  none:
```

Comment:
There is no need to describe the resources which will not have permissions. This is the default one, when there are overrides.